core.lua

```diff
@@ -189,6 +191,7 @@ do
     ns.LOOKUP_MAX_SIZE = floor(2^18-1) -- the maximum index we can use in a table before we start to get errors
     ns.CURRENT_SEASON = 1 -- the current mythic keystone season. dynamically assigned once keystone data is loaded.
     ns.RAIDERIO_ADDON_DOWNLOAD_URL = "https://rio.gg/addon"
+    ns.RAIDERIO_DOMAIN = IS_CLASSIC and "classic.raider.io" or "raider.io"
```

Comment:
The domains will need to be modified to match the specific site:

WOW_PROJECT_MAINLINE => `raider.io`
WOW_PROJECT_WRATH_CLASSIC => `classic.raider.io`
WOW_PROJECT_CLASSIC => `era.raider.io`

Since we can expect there will be future versions of classic, we may want to be able to bundle the classic checks. So it's:

WOW_PROJECT_CLASSIC ==> era
WOW_PROJECT_WRATH_CLASSIC || WOW_PROJECT_BURNING_CRUSADE_CLASSIC ==> classic

Reply:
I expanded the line to select the appropriate domain based on the `IS_CLASSIC` and `IS_WRATH` checks.